qiskit/aqua/circuits/phase_estimation_circuit.py

```diff
@@ -129,7 +130,13 @@ def construct_circuit(
                 if self._operator is not None:
                     q = QuantumRegister(self._operator.num_qubits, name='q')
                 elif self._unitary_circuit_factory is not None:
-                    q = QuantumRegister(self._unitary_circuit_factory.num_target_qubits, name='q')
+                    if isinstance(self._unitary_circuit_factory, CircuitFactory):
```

Comment:
Could we change the method signature so the types are now typehints rather than in the docstring since this is being edited. This is partly motivated since it seems that the unitary_circuit_factory may be other than a CircuitFactory type now and the latter is all that is stated as the type.

Reply:
Good point, added in 69175a230

Reply:
This functionality has been removed in 1adaed0 as it wasn't needed anymore 👍